node_modules/app/format_date.js

```diff
@@ -1,24 +1,39 @@
 'use strict'
 
+var S = require('sanctuary')
+var R = require('ramda')
+
 module.exports = function (d) {
   var date = format_date(d)
-  if (date[1] == null) return date[0]
-  return date[1]
+  return S.either(S.I, S.I)(date)
+}
+
+module.exports.date_only = S.pipe(
+  [ check_date
+  , date_only
+  , S.either(S.I, S.I)
+   ]
+)
+
+function date_only (d) {
+  return d.map(date => date.split('T')[0])
 }
 
 function format_date (d) {
-  var date = new Date(d) // allows arg to be date string
-  if (!d || is_bad_date(date)) return [ d, null ]
-  date.setHours(12) // hack because of weirdness with times, sails and waterline
-  var dateString = date.toISOString().split('T')[0]
-  return [ null,  dateString.split('-').reverse().join('/') ]
+  var date = check_date(d) // allows arg to be date string
+  date.map(d => date_to_string((new Date(d)).setHours(12))) // hack because of weirdness with times, sails and waterline
+  date = date_only(date)
```

Comment:
This reassignment rather than mutation and its pretty lazy, but because this variable is only used internally, I don't have a huge issue with it.

Actually ignore my previous comment and let's have

```js
date = date_only(date.map(d => date_to_string((new Date(d)).setHours(12))))
```